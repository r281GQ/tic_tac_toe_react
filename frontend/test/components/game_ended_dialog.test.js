import React from 'react';
import { shallow } from 'enzyme';
import GameEndedDialog from './../../src/components/game_ended_dialog';

describe('calorie_target_result_sum_label', () => {
  it('should render', () => {
    const component = shallow(
      <GameEndedDialog
        createNewGameHandler={value => value}
        isModalOpen={false}
      />
    );
    expect(component).toHaveLength(1);
  });
});
