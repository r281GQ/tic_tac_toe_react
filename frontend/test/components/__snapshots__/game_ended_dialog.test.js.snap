// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calorie_target_result_sum_label should render 1`] = `
<Dialog
  actions={
    Array [
      <FlatButton
        disabled={false}
        fullWidth={false}
        label="new game"
        labelPosition="after"
        labelStyle={Object {}}
        onClick={[Function]}
        onKeyboardFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onTouchStart={[Function]}
        primary={false}
        secondary={false}
    />,
    ]
  }
  autoDetectWindowHeight={true}
  autoScrollBodyContent={false}
  modal={true}
  open={false}
  repositionOnUpdate={true}
  title="Tic Tac Toe"
/>
`;
